src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -149,7 +131,36 @@ class CalculatorForm extends React.Component {
           ]}
           value={this.props.inputs.yellowRibbonRecipient}
           onChange={this.handleInputChange}/>
-        {amountInput}
+        { this.props.inputs.yellowRibbonRecipient === 'yes' ?
```

Comment:
This follows the pattern of the rest of the Dropdowns in this app.

Reply:
I think you could use the `ExpandingGroup` component from formation to implement the UI from the mock.

Reply:
There's some CSS hacking that'll have to be done to be able to use `ExpandingGroup` here. `.form-expanding-group-open` style messes this up.

![screen shot 2018-06-14 at 12 58 00](https://user-images.githubusercontent.com/4998130/41432237-98053fa6-6fd2-11e8-8a6c-c16b4c8dbd90.png)

These styles are added to the outer div- I could override them with a new class passed into the `additionalClass` prop but that gets applied to an inner div. 
![screen shot 2018-06-14 at 13 02 23](https://user-images.githubusercontent.com/4998130/41432396-3e5de358-6fd3-11e8-8ef2-d3460ebf2393.png)